modules/csharp/examples/code_snippets/Program.cs

```diff
@@ -1811,7 +1811,9 @@ private void CreateArrayIndex()
 
             {
                 // tag::array-index-single[]
+                // tag::array-index-config[]
```

Comment:
just to confirm: for dotNet we don't have a script to generate this Program.cs (e.g. like we do for Android/Java)?

e.g. want to be sure this won't get overwritten if the file gets regenerated.